src/python_testing/matter_testing_infrastructure/matter/testing/matter_testing.py

```diff
@@ -419,7 +422,10 @@ async def commission_devices(self) -> bool:
 
         return await commission_devices(dev_ctrl, dut_node_ids, setup_payloads, commissioning_info)
 
-    async def open_commissioning_window(self, dev_ctrl: Optional[ChipDeviceCtrl.ChipDeviceController] = None, node_id: Optional[int] = None, timeout: int = 900) -> CustomCommissioningParameters:
+    async def open_commissioning_window(self,
```

Comment:
Can you please back out the formatting changes in this file so the diff is easier to read? I think this is just formatting...